lib/internal/streams/async_iterator.js

```diff
@@ -86,6 +86,20 @@ const ReadableStreamAsyncIteratorPrototype = Object.setPrototypeOf({
       return Promise.resolve(createIterResult(null, true));
     }
 
+    if (this[kStream].destroyed) {
+      // this is needed because if .destro(err) is called, the error
+      // will be emitted via nextTick
+      return new Promise((resolve, rejects) => {
```

Comment:
any reason for `rejects` rather than `reject`?

Reply:
none, I’ll fix it.